src/transitions/style.ts

```diff
@@ -20,13 +20,12 @@ export default css`
   }
 
   :global(.slide-up) {
-    transition: ease-in-out ${transition.duration.base};
-    transition-property: opacity, transform;
+    transition: all ${transition.duration.base} ease-in-out;
```

Comment:
Why put `all` here instead of specifying the different properties as it was before?